[2021-04-14 13:28:35,176] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop 2021-03-31T00:00:00+00:00 [queued]>
[2021-04-14 13:28:35,221] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop 2021-03-31T00:00:00+00:00 [queued]>
[2021-04-14 13:28:35,222] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-14 13:28:35,223] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-04-14 13:28:35,223] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-14 13:28:35,280] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): run_after_loop> on 2021-03-31T00:00:00+00:00
[2021-04-14 13:28:35,301] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', '2021-03-31T00:00:00+00:00', '--job-id', '40', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpv18uf4ha', '--error-file', '/tmp/tmp2051tael']
[2021-04-14 13:28:35,305] {standard_task_runner.py:77} INFO - Job 40: Subtask run_after_loop
[2021-04-14 13:28:35,289] {standard_task_runner.py:52} INFO - Started process 99 to run task
[2021-04-14 13:28:35,596] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.run_after_loop 2021-03-31T00:00:00+00:00 [running]> on host 33dac7d1aa5e
[2021-04-14 13:28:35,785] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2021-03-31T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-31T00:00:00+00:00
[2021-04-14 13:28:35,788] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-04-14 13:28:35,791] {bash.py:158} INFO - Running command: echo 1
[2021-04-14 13:28:35,820] {bash.py:169} INFO - Output:
[2021-04-14 13:28:35,832] {bash.py:173} INFO - 1
[2021-04-14 13:28:35,836] {bash.py:177} INFO - Command exited with return code 0
[2021-04-14 13:28:36,008] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20210331T000000, start_date=20210414T132835, end_date=20210414T132836
[2021-04-14 13:28:36,142] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-04-14 13:28:36,183] {local_task_job.py:146} INFO - Task exited with return code 0
